article/Experiment/performance-comparisons.tex

```diff
@@ -0,0 +1,43 @@
+\subsection{Performance Comparison with State-of-the-Arts}
+\autoref{tab:baselines-ndcg} and \autoref{tab:baselines-recall} shows the results of our experiments with NDCG@50 and Recall@50 respectively.
+We perform comparisons to LightGCN, NGCF, GC-MC and GCN.
+For all methods 5 convolutions were used. For Amazon-Book 3 convolutions performed better, but as ALC and BLC were calculated using 5 convolutions we compare it to 5 convolutions.
+ALC and BLC possibly also perform better on Amazon-Book with fewer convolutions.
+The primary observations from the experiment are:
+\begin{itemize}
+    \item ALC, BLC and $\mathbf{e}^{(i)}$ generally perform better than all other methods, except on the datasets Amazon-Cell-Electronic and Amazon-Cloth-Sport. Amazon-Cell-Electronic diverges from the other datasets, as it has a item/user ratio of 17.42, where the others have a item/user ration below 8. Amazon-Cloth-Sport is similar to Amazon-Cell-Sport in terms of item/user ration, however it has twice as many users and items.
```

Comment:
```suggestion
    \item ALC, BLC and $\mathbf{e}^{(i)}$ generally perform better than all other methods, except for on Amazon-Cell-Electronic and Amazon-Cloth-Sport. Amazon-Cell-Electronic differentiates from other datasets because it has an item/user ratio of 17.42, while other datasets have an item/user ratio below 8. Amazon-Cloth-Sport is similar to Amazon-Cell-Sport in terms of item/user ratio, however, it has twice as many users and items.
```